src/kagglehub/handle.py

```diff
@@ -133,3 +145,13 @@ def parse_model_handle(handle: str) -> ModelHandle:
 
     msg = f"Invalid model handle: {handle}"
     raise ValueError(msg)
+
+
+def parse_competition_handle(handle: str) -> CompetitionHandle:
+    parts = handle.split("/")
+
+    if len(parts) == 1:
+        return CompetitionHandle(owner="", competition=parts[0])
```

Comment:
Given that `owner` is not common anymore to all handles, can you move it out of the `ResourceHandle` parent class? And add it to DatasetHandle and ModelHandle.

Reply:
thanks, much cleaner